src/marker-window.c

```diff
@@ -119,22 +122,30 @@ show_unsaved_documents_warning (MarkerWindow *window)
     dialog = gtk_message_dialog_new_with_markup(GTK_WINDOW (window),
                                                 GTK_DIALOG_MODAL,
                                                 GTK_MESSAGE_QUESTION,
-                                                GTK_BUTTONS_OK_CANCEL,
+                                                GTK_BUTTONS_NONE,
                                                 "<span weight='bold' size='larger'>"
                                                 "Discard changes to the document?"
                                                 "</span>\n\n"
                                                 "The document has unsaved changes "
                                                 "that will be lost if it is closed now.");
+
+    save_or_save_as = "Save as";
   }
 
+  gtk_dialog_add_buttons(dialog,
```

Comment:
You can use GTK_DIALOG to fix this as i did in #332 
https://github.com/fabiocolacio/Marker/pull/332/commits/a926e22386b8977b0a12da7bc69900d3923fcfcd#diff-eb21b5e37df33cd6a3d1bc7a89cbf093dd322d54b982dcf2cf29ec37c4f1dde6R132